### PR TITLE
Make SignInComponent to fire backend requests

### PR DESCRIPTION
**Pull request by tirr-c**
## TODO
- [x] Make UserEffects to request to the backend
- [x] Write some tests
- [x] Wait for #16

### CODE REVIEW

File: frontend/src/app/app.module.ts [hunk 1]
```diff
@@ -35,6 +38,7 @@ import { UserService } from './user.service';
   providers: [
     UserService,
     { provide: APP_BASE_HREF, useValue: '/' },
+    { provide: XSRFStrategy, useFactory: xsrfFactory },
```
Comment (skystar-p) [inline frontend/src/app/app.module.ts]: ㅎㄷㄷ

**Comment by skystar-p** [comment]
@web-mighty-homu r+

**Comment by web-mighty-homu** [comment]
:clipboard: Looks like this PR is still in progress, ignoring approval

**Comment by web-mighty-homu** [comment]
:clipboard: Looks like this PR is still in progress, ignoring approval

**Comment by skystar-p** [comment]
@web-mighty-homu r+

**Comment by web-mighty-homu** [comment]
:pushpin: Commit 1c09ba5 has been approved by `skystar-p`

**Comment by web-mighty-homu** [comment]
:hourglass: Testing commit 1c09ba59c5387613822db43899b5d28dd5227cbf with merge 0604b573ca101f68b032fc083ef0c68083f46299...

**Comment by web-mighty-homu** [comment]
:sunny: Test successful - [status-travis](https://travis-ci.org/swsnu/swpp17-team2/builds/300845840?utm_source=github_status&utm_medium=notification)
Approved by: skystar-p
Pushing 0604b573ca101f68b032fc083ef0c68083f46299 to master...

**Pull request merged**